packages/server/src/api/routes/tests/queries/generic-sql.spec.ts

```diff
@@ -34,12 +37,14 @@ const createTableSQL: Record<string, string> = {
 const insertSQL = `INSERT INTO test_table (name) VALUES ('one'), ('two'), ('three'), ('four'), ('five')`
 const dropTableSQL = `DROP TABLE test_table;`
 
-describe.each([
-  ["postgres", databaseTestProviders.postgres],
-  ["mysql", databaseTestProviders.mysql],
-  ["mssql", databaseTestProviders.mssql],
-  ["mariadb", databaseTestProviders.mariadb],
-])("queries (%s)", (dbName, dsProvider) => {
+describe.each(
+  [
+    DatabaseName.POSTGRES,
+    DatabaseName.MYSQL,
+    DatabaseName.SQL_SERVER,
+    DatabaseName.MARIADB,
+  ].map(name => [name, getDatasource(name)])
+)("queries (%s)", (dbName, dsProvider) => {
```

Comment:
I changed the way this works for 2 reasons:

1. It's a slightly simpler API, don't have to worry about stopping containers anymore.
2. When you run this test locally, it actually starts up all containers straight away now instead of starting them just before the test runs. This results in faster runtime for tests spinning up multiple containers.